lib/models/basic-property.js

```diff
@@ -16,6 +16,9 @@ class BasicProperty extends Base {
             case BasicTypes.Text:
                 this.defaultValue = "'text'";
                 break;
+            case BasicTypes.Password:
```

Comment:
If you are putting all the `case` together on line 44, maybe it should also be the case here.